README.md

```diff
@@ -15,23 +15,26 @@ pip3 install -r requirements.txt
 ```
 ## Prepare environment variables
 
-Inside the `.env` file, override the default variables with your [JWT token](https://api.exchange.bullish.com/docs/api/rest/#overview--generate-a-jwt-token) 
-and [API key](https://api.exchange.bullish.com/docs/api/rest/#overview--generate-an-api-key):
+Scripts in this repository rely on environment variables.
+
+Those have to be supplied via an `.env` file in the below format. [Here](.env)
+
 ````
 export BX_JWT=< your credential >
-
 export BX_PUBLIC_KEY=< your credential >
 export BX_PRIVATE_KEY=< your credential >
 export BX_API_METADATA=< your credential >
 export BX_AUTHORIZER=< your credential >
 export BX_TRADING_ACCOUNT_ID=< your credential >
 ````
 
-## Scripts
+More info on how to obtain them is available in [this guide](https://api.exchange.bullish.com/docs/api/rest/#overview--how-to-send-authenticated-requests).  
+
+Specifically,
+
+- [API key (Public Key, Private Key, API Metadata)](https://api.exchange.bullish.com/docs/api/rest/#overview--generate-an-api-key):
+
+- [Login Tokens (JWT token and Authorizer)](https://api.exchange.bullish.com/docs/api/rest/#overview--generate-a-jwt-token) 
 
-For a full walk-through on the usage of these scripts, start [here](https://api.exchange.bullish.com/docs/api/rest/#overview--how-to-send-authenticated-requests).  
```

Comment:
I moved and reworded it here

https://github.com/bullish-exchange/api-examples/pull/18/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R31